registration-processor/core-processor/registration-processor-reprocessor-stage/src/main/java/io/mosip/registration/processor/reprocessor/stage/WorkflowEventUpdateVerticle.java

```diff
@@ -96,7 +96,7 @@ public void consume(MosipEventBus mosipEventBus, MessageBusAddress fromAddress)
 			vertx.executeBlocking(future -> {
 				MDC.setContextMap(mdc);
 				JsonObject jsonObject = (JsonObject) msg.getBody();
-				WorkFlowEventDTO workFlowEventDTO = jsonObject.mapTo(WorkFlowEventDTO.class);
+				WorkflowEventDTO workFlowEventDTO = jsonObject.mapTo(WorkflowEventDTO.class);

```

Comment:
object name also use workflow with small f, we can do a simple case sensite text search and replace